### PR TITLE
Remove rate limit tests that have become flaky.

### PR DESCRIPTION
**Pull request by bbaker6225**
Credentials Pull Request
---

Make sure that the following steps are done before rebasing/merging

  - [ ] Request a review if desired
  - [ ] Squash/Fixup your branch to one commit
  - Config/Dependency Changes (e.g. config files, scripts that are used for provisioning etc.)
    - [ ] Make sure you have updated [edx/configuration](https://github.com/edx/configuration) and [edx/devstack](https://github.com/edx/devstack) if necessary
    - [ ] Make sure the change builds successfully in a sandbox
  - UI Changes 
    - [ ] Consider other browsers (e.g. Firefox)
    - [ ] Check mobile view
    - [ ] Consider accessibility (e.g. Run aXe on the page)

### CODE REVIEW

**Comment by BenjiLee** [comment]
@bbaker6225 Is there a way to expliclty set the rate limit to 1? That way we can test expected behaviour when it does get hit.

**Comment by bbaker6225** [comment]
@BenjiLee We can override the rate limit in the tests, but since they're hitting the actual endpoint they will still use the limit that is set on the actual classes, which I don't think we found a way to override when we tried to do this the first time.

File: credentials/apps/api/v2/tests/test_views.py [hunk 1]
```diff
@@ -503,123 +500,3 @@ def test_update(self, method):
...
-        """ Login as the given user. """
-        self.client.logout()
-        self.client.login(username=user.username, password=USER_PASSWORD)
-
-    def add_user_permission(self, user, permission):
-        """ Assigns a permission of the given name to the user. """
-        user.user_permissions.add(Permission.objects.get(codename=permission))
-
-    @ddt.data('credential_view', 'grade_view', 'staff_override')
-    def test_throttle_configuration(self, scope):
```
Comment (BenjiLee) [inline credentials/apps/api/v2/tests/test_views.py]: We need to keep this test to verify that there is a rate limit set.

Comment (bbaker6225) [reply credentials/apps/api/v2/tests/test_views.py]: Added back in

Comment (BenjiLee) [reply credentials/apps/api/v2/tests/test_views.py]: Do we need a similar test for the other endpoint Records?

Comment (bbaker6225) [reply credentials/apps/api/v2/tests/test_views.py]: We couldn't find a good way to do it last time, since the other endpoints weren't DRF

**Pull request merged**